modules/operator-upgrade-modify-central-custom-resource-tech-preview.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_module-type: PROCEDURE
+[id="operator-upgrade-modify-central-custom-resource-tech-preview_{context}"]
+= Modifying Central custom resource for external database
+
+:FeatureName: External PostgreSQL support
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* Provision a database that supports PostgreSQL 13 that will be used solely for {product-title-short}.
+
+[NOTE]
+====
+* A multi-tenant database is currently unsupported.
+* Superuser role is required with permissions to create and delete databases.
+* PgBouncer is currently unsupported.
+====
+
+.Procedure
+. Create a password secret in the deployed namespace by navigating to the *Workloads* -> *Secrets* page on the {ocp} web console. Create a `Key/Value secret` with the key `password` and the value of a plaintext file containing the password for the superuser of the provisioned database.
```

Comment:
>Create a `Key/Value secret` with the key `password` and the value of a plaintext file containing the password for the superuser of the provisioned database.

Is this 
- Create a `Key/Value secret` with the key `password` and the value as the path of a plaintext file containing the password for the superuser of the provisioned database. ?

If not can you explain.